float32[] distribution
geometry_msgs/PoseArray waypoints